config/application.rb

```diff
@@ -21,6 +21,7 @@ class Application < Rails::Application
     # config.i18n.default_locale = :de
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
+    config.action_mailer.default_url_options = { host: 'https://rustworks-schedlr.herokuapp.com/rsvp' }
```

Comment:
Why does this end with `/rsvp` ? The host should only be asking for the subdomain and domain. Like this:
`config.action_mailer.default_url_options = { :host => 'yourapp.heroku.com' }`